airflow/providers/google/cloud/operators/gcs.py

```diff
@@ -536,8 +536,10 @@ class GCSFileTransformOperator(BaseOperator):
     data from source, transform it and write the output to the local
     destination file.
 
-    :param source_bucket: The key to be retrieved from GCS. (templated)
-    :param destination_bucket: The key to be written from GCS. (templated)
+    :param source_bucket: The bucket to locate the source_object. (templated)
+    :param source_object: The key to be retrieved from GCS. (templated)
+    :param destination_bucket: The bucket to upload the key after transformation. (templated)
+    :param destination_object: The key to be written in GCS. (templated)
```

Comment:
To me, it seems better to be explicit rather than implicit here since this will appear in the Python API documentation rather than having to read the source code to figure it out.